lib/hackerone/client/incremental/activities.rb

```diff
@@ -0,0 +1,101 @@
+module HackerOne
```

Comment:
For this file in general, I think the tiny methods might work better as inline code with a comment rather than a separate one line method that's only called in one place. I *think* avoiding ivars in favor of locals will help readability. Double emphasis on *think*.